### DISCUSSION TITLE
Is there a way to center cursor line on screen?

### DISCUSSION THREAD

**Discussion by brecke** [LazyVim/LazyVim#1079] [Q&A] [answered]
Previously on nvim I would press ZZ and that would center the current line vertically on screen, but I just noticed it doesn't work on lazyVim. Am I missing something?

**Comment by dpetka2001** [comment, marked as answer]
It's actually `zz`. `ZZ` is used to write and quit Neovim. See `:h ZZ` and `:h zz`.

**Comment by brecke** [comment]
you're right. 🤦🏻‍♂️
